src/Collection/Iterator/CompositeIterator.php

```diff
@@ -35,13 +35,13 @@ public function __construct(array $iterators)
     /**
      * @template U
      *
-     * @param Iterator<U> ...$iterators
+     * @param Iterator<U> ...$elements
      *
      * @return Iterator<U>
      */
-    public static function of(...$iterators): Iterator
+    public static function of(...$elements): Iterator
```

Comment:
This was reported by psalm, I am not sure if the argument name should be changed error ignored.